src/abTestingInfra/permutations.ts

```diff
@@ -0,0 +1,166 @@
+let fs = null;
+
+if (typeof window === "undefined") {
+  fs = require("fs-extra");
+}
+
```

Comment:
Any chance we can do this immediately after the `import`s instead?

Reply:
Done